bigtable/metrics_monitoring_exporter.go

```diff
@@ -86,31 +88,41 @@ func newMonitoringExporter(ctx context.Context, project string, opts ...option.C
 		client:    client,
 		shutdown:  make(chan struct{}),
 		projectID: project,
+		errLogger: errLogger,
 	}, nil
 }
 
+func (me *monitoringExporter) logErrorAndReturn(src string, err error) error {
+	if err != nil && me.errLogger != nil {
+		me.errLogger.Println(fmt.Errorf("%v: %w", src, err).Error())
+	}
+	return err
+}
+
 // ForceFlush does nothing, the exporter holds no state.
-func (e *monitoringExporter) ForceFlush(ctx context.Context) error { return ctx.Err() }
+func (me *monitoringExporter) ForceFlush(ctx context.Context) error {
+	return me.logErrorAndReturn("ForceFlush", ctx.Err())
```

Comment:
generally logging + return errors is an anti pattern,  I think you should revert this to just return an error

Reply:
wrapping and returning error